app/services/user_signup.rb

```diff
@@ -0,0 +1,52 @@
+class UserSignup
+
+  attr_reader :error_message
+
+  def initialize(user)
+    @user = user
+  end
+
+  def sign_up(stripe_token, invitation_token)
```

Comment:
`invitation_token` should be optional here. It will create better interface also for testing and not every sign up needs to have the invitation token.